### PR TITLE
Add configuration for cheat codes

### PR DESCRIPTION
**Pull request by anvacaru**
Created a new configuration named `<cheatcodes/>` which contains cells that are used during the execution of various cheatcodes.
Currently it only holds cells for cheatcodes involving `pranks`.
This configuration, together with the one of `<kevm/>` will form the main configuration of the project, `<foundry/>`.

### CODE REVIEW

**Review by tothtamas28 — APPROVED** [review]
LGTM. Please wait for review from @ehildenb as well.

**Review by ehildenb — APPROVED** [review]
Looks nice to me as well!

**Pull request merged**
